lib/build.sh

```diff
@@ -1,5 +0,0 @@
-
-# build static html pages
-bundle install
-npm install
-bundle exec middleman build
```

Comment:
Do folks still use any of these non-release-related scripts when building locally?

Reply:
the local build steps are documented at the start of the `README`

Reply:
Hmmm ok. l guess for this repo it's unlikely anyone will have e.g a local shim relying on this shortcut being here...